filepathfilter/filepathfilter.go

```diff
@@ -138,15 +148,19 @@ func NewPattern(p string) Pattern {
 			}
 		}
 	}
-	tracerx.Printf("filepathfilter: rewrite %q as %q", p, pp)
+
+	msg := fmt.Sprintf("filepathfilter: rewrite %q as %q", p, pp)
+	if useLit {
+		msg = fmt.Sprintf("%s (or fallback)", msg)
```

Comment:
Should we prepend this with `filepathfilter:` as well like above?

Reply:
This already happens since the argument that we're filling in for the first `%s` is the `msg` as initialized above.